zds/settings/abstract_base/zds.py

```diff
@@ -77,7 +77,8 @@
             'fee': zds_config.get('association_fee', '20 €'),
             'email': 'zestedesavoir@gmail.com',
             'email_ca': 'ca-zeste-de-savoir@googlegroups.com',
-            'forum_ca_pk': 25
+            'forum_ca_pk': 25,
+            'suscribe_link': 'https://www.helloasso.com/associations/zeste-de-savoir/adhesions/zeste-de-savoir-cotisations-2018'
```

Comment:
Correction du mot "subscribe" et utilisation de `zds_config` pour pouvoir changer la configuration sans devoir attendre une nouvelle version de ZDS

```suggestion
            'subscribe_link': zds_config.get('association_subscribe_link', 'https://www.helloasso.com/associations/zeste-de-savoir/adhesions/zeste-de-savoir-cotisations-2018')
```